manifests/params.pp

```diff
@@ -39,15 +39,23 @@
           $plugin_config_dir           = '/etc/collectd/managed_config'
           $collectd_config_file        = '/etc/collectd/collectd.conf'
           $filtering_config_file       = '/etc/collectd/filtering_config/filtering.conf'
-          $mysql_socket_file           = '/var/lib/mysqld/mysqld.sock'
+          $bundled_plugins             = [
+                                            'collectd-apache',
+                                            'collectd-kafka',
```

Comment:
thanks for making the fix Jake.  This bundled_plugins needs to be better defined as debian and rhel collectd installation have different sets of bundled plugins (name change).  Also the previous fix you made to gate the plugin package installation on debian combines this list with logic to not install packages on debian specifically (might be broken for some plugins).